package.json

```diff
@@ -4,7 +4,7 @@
   "version": "0.0.1",
   "license": "MIT",
   "scripts": {
-    "prestart": "cd packages/components && npm run build:cssTypes",
```

Comment:
any reason to keep this?

Reply:
@darryltec i don't think so.  Might be prudent to also remove that script in `packages/components` when we're relying on the plugin to build our CSS types anyways

Reply:
I'm waiting to see from circle if we actually need it.  Need some advise on how to confirm this in local as well.